packages/next/build/webpack/loaders/next-client-pages-loader.ts

```diff
@@ -0,0 +1,29 @@
+import {loader} from 'webpack'
+import loaderUtils from 'loader-utils'
+
+export type ClientPagesLoaderOptions = {
+  absolutePagePath: string,
+  page: string
+}
+
+const nextServerlessLoader: loader.Loader = function () {
```

Comment:
This loader replaces both `pages-plugin.js` and the `hot-self-accept-loader`. It's no longer a "self-accept" but an accept for the specific page, which is pretty much the same as what it used to do, except if an error is thrown it bubbles up correctly in this case.